client/app/lib/components/redux-form/Toggle.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import MaterialToggle from 'material-ui/Toggle';
+import createComponent from './createComponent';
+import mapError from './mapError';
+
+const errorStyle = {
+  color: 'red',
+};
+
+// Toggle implementation with an error displayed at the bottom.
+const Toggle = function ({ errorText, ...props }) {
+  return (
+    <div>
+      <MaterialToggle {...props} />
+      {
+        errorText &&
+        <div style={errorStyle}>{errorText}</div>
+      }
+    </div>
+  );
+};
+
+Toggle.propTypes = {
+  errorText: PropTypes.string,
+};
+
+export default createComponent(
```

Comment:
Hmm, I think it's unnecessary because the usage is exactly same as the original material UI component, you can find an example [here](https://github.com/Coursemology/coursemology2/blob/master/client/app/bundles/course/assessment/containers/AssessmentForm/index.jsx#L130).

The code here is for integrating with redux-form, so it can be put in `Field`.